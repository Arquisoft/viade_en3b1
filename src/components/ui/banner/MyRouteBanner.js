import React from 'react'
import { Grid, CssBaseline, makeStyles, Typography } from '@material-ui/core';

export default function MyRouteBanner() {
    const classes = useStyles();

    return (
        <div>
            {/* https://source.unsplash.com/user/pablocanalsuarez/likes/1600x900 */}
            {/* https://source.unsplash.com/collection/9828433/1600x900 */}
            <Grid container component="main" className={classes.root}>
                <CssBaseline />
                <Grid
                    item
                    xs={false}
                    sm={12}
                    md={12}
                    className={classes.image}>
                    <Grid container spacing={0}>

                        <Grid item sm={12} md={12}>
                            <Typography className={classes.text}>Your routes,</Typography>
                        </Grid>

                        <Grid item sm={12} md={12}>
                            <Typography className={classes.text2}>your rules.</Typography>
                        </Grid>
                        
                        {/* <Typography className={classes.text}>Your routes,</Typography>
                        <Typography className={classes.text2}>your rules.</Typography> */}
                    </Grid>
                </Grid>
            </Grid>

        </div>
    )
}

const useStyles = makeStyles((theme) => ({

    root: {
        // height: '60em',
        height: '30vw',
    },
    image: {
        backgroundImage: `url(https://source.unsplash.com/collection/9828433/1600x900)`,
        backgroundRepeat: 'no-repeat',
        backgroundColor:
            theme.palette.type === 'light' ? theme.palette.grey[50] : theme.palette.grey[900],
        backgroundSize: 'cover',
        backgroundPosition: 'center',
    },
    text: {
        marginLeft: '4vw',
        fontSize: '12vw',
        color: '#ffffff'
    },
    text2: {
        marginLeft: '6.5vw',
        fontSize: '7vw',
        color: '#ffffff'
    }
}));

