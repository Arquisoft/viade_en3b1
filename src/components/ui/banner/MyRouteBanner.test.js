import MyRouteBanner from './MyRouteBanner';

test("Define MyRouteBanner", () => {
    expect(MyRouteBanner).toBeDefined();
});

test("Renders correctly", () => {
    const wrapper = render(<MyRouteBanner />);
    expect(wrapper).toMatchSnapshot();
});