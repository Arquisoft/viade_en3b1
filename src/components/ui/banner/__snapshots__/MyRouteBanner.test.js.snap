// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<div>
  <main
    class="MuiGrid-root makeStyles-root-1 MuiGrid-container"
  >
    <div
      class="MuiGrid-root makeStyles-image-2 MuiGrid-item MuiGrid-grid-sm-12 MuiGrid-grid-md-12"
    >
      <div
        class="MuiGrid-root MuiGrid-container"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-sm-12 MuiGrid-grid-md-12"
        >
          <p
            class="MuiTypography-root makeStyles-text-3 MuiTypography-body1"
          >
            Your routes,
          </p>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-sm-12 MuiGrid-grid-md-12"
        >
          <p
            class="MuiTypography-root makeStyles-text2-4 MuiTypography-body1"
          >
            your rules.
          </p>
        </div>
      </div>
    </div>
  </main>
</div>
`;
